app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -33,8 +32,7 @@ class MS.Views.CalendarView extends Backbone.View
       # We use a 1ms timeout so that fullCalendar can finish drawing the event in the new calendar cell.
       setTimeout =>
         if confirm(I18n.t("loan.move_date_confirm.body"))
-          projectId = @$el.find('#calendar').data('project-id')
```

Comment:
I'm curious why this was ever here instead of using `event.model_id`. Perhaps just an oversight. In any case if this works then great! As long as we have tested it in all the various calendar locations.